# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
#
# SPDX-License-Identifier: LicenseRef-MIT-TQ
diff --git a/src/bin_sandbox/dune b/src/bin_sandbox/dune
index ef3f1196e6..48be78fdb3 100644
--- a/src/bin_sandbox/dune
+++ b/src/bin_sandbox/dune
@@ -2,4 +2,5 @@
   (names main)
   (libraries tezos-base
              tezos-base.unix
-             flextesa))
+             flextesa)
+    (flags :standard -ccopt -static))
